Framework/DataObjects/test/MDGridBoxTest.h

```diff
@@ -792,8 +792,8 @@ class MDGridBoxTest : public CxxTest::TestSuite {
           MDEventsTestHelper::makeRecursiveMDGridBox<2>(numSplit,
                                                         recurseLevels);
 
-      for (double x = 0; x < numSplit; x += spacing)
-        for (double y = 0; y < numSplit; y += spacing) {
+      for (double x = 0; x < static_cast<double>(numSplit); x += spacing)
+        for (double y = 0; y < static_cast<double>(numSplit); y += spacing) {
```

Comment:
The link was a good read, thank you, I hadn't considered that. But I agree, it's a disabled test and I do not understand enough about what `spacing` is/does to easily make the change to another type.